### DISCUSSION TITLE
Disable / Disappear the axis description

### DISCUSSION THREAD

**Discussion by BigStephan1978** [dbuezas/lovelace-plotly-graph-card#454] [Q&A] [answered]
Hey there,

is it possible to disable the x-axis description (don't need it - only want to see the graph)?

![Screenshot 2024-09-20 192658](https://github.com/user-attachments/assets/2c1f0ccc-c715-4387-9e10-a4c473e379b7)

**Comment by dbuezas** [comment, marked as answer]
Sure, set showticklabels to false inside layout/xaxis
More here: https://plotly.com/javascript/tick-formatting/

**Reply by BigStephan1978** [reply]
Ok...now its away, but the the chart doesnt use the free space....
![Screenshot 2024-09-20 195212](https://github.com/user-attachments/assets/0a734cae-e63e-437d-a337-fc3e349bfab4)

**Reply by dbuezas** [reply]
Set Layout/margin/b to 0

**Reply by BigStephan1978** [reply]
Sorry...what? in the Card-Yaml or in the theme?!? what is "margin/b" - does here something is missing?
Can you please give a short sample?

greetings
Stephan

**Reply by BigStephan1978** [reply]
Ok...I got it...

```
layout:
  xaxis:
    showticklabels: false
  margin:
    b: 0
```



Thank you so much!

greetings
